.github/workflows/pipeline.yml

```diff
@@ -8,7 +8,9 @@ on:
 jobs:
   check-newsfile:
     name: Check PR has a changelog
-    if: ${{ github.base_ref == 'main' || contains(github.base_ref, 'release-') }}
+    if: >
+      (github.base_ref == 'main' || contains(github.base_ref, 'release-'))
+      && github.event.pull_request.user.login != 'dependabot[bot]'
```

Comment:
I agree with these changes, but can we move them out into a separate PR? The title of this PR seems like it would hide this change in historical records.

Reply:
(Though this will fail CI until that is merged)